filebeat/input/kafka/config.go

```diff
@@ -215,7 +216,6 @@ func newSaramaConfig(config kafkaInputConfig) (*sarama.Config, error) {
 	)
 
 	if err := k.Validate(); err != nil {
-		logp.Err("Invalid kafka configuration: %v", err)
```

Comment:
:tada: :tada: :tada:

Reply:
This is good I think / hope? :sweat_smile:

Reply:
Yes :D